src/yamlToAnalytics.ts

```diff
@@ -137,5 +137,10 @@ export default async (inDir: string, outFile: string) => {
     properties,
 });
 `));
-    fs.writeFileSync(outFile, file.join("\n"));
+    return file.join("\n");
+};
+
+export default async (inDir: string, outFile: string) => {
```

Comment:
Could we give this a proper name for verbosity (`export const ymlFilesToTypeFile`?)

Reply:
Yes we could! I don't think one can `export default` a `const` though, it would then have to be `export default function ymlFilesToTypeFile(...)`.

Reply:
And it's Mike's repo so I wouldn't change his default exports ;)